pages/annotation.vue

```diff
@@ -1,68 +1,58 @@
 <template>
 
-    <b-container fluid>
-
-        <!-- Navigation bar -->
-		<tool-navbar 
-			:navItems="pageData"
-			:navOrder="pageOrder"
-			:pageName="pageData.annotation.fullName">
-		</tool-navbar>
-
-        <b-row>
-            <b-col cols="4"></b-col>
-            <b-col cols="4">
-                <img class="vertical-center-mockup"
-                     :src="require(`~/assets/${mockupImage.filename}`)"
-                     :width="mockupImage.width"
-                     :height="mockupImage.height"/>
-            </b-col>
-            <b-col cols="4"></b-col>
-        </b-row>        
-
-    </b-container>
+  <b-container fluid>
+
+    <!-- Navigation bar -->
+    <tool-navbar
+      :navItems="pageData"
+      :navOrder="pageOrder"
+      :pageName="pageData.annotation.fullName">
+    </tool-navbar>
+
+    <!-- This gives us built-in keyboard navigation! -->
+    <b-tabs pills card vertical>
+      <!--      TODO: hardcode the pages and just toggle visibility based on state-->
+      <b-tab v-for="page in pages" :title="page.title" :key="page.id">
+        <b-card-text>
+          <component
+            :is="page.component"
+          ></component>
+        </b-card-text>
+      </b-tab>
+    </b-tabs>
+  </b-container>
 
 </template>
 
 <script>
-
-	// Allows for reference to store data by creating simple, implicit getters
-	import { mapState } from "vuex";
-
-    export default {
-
-        data() {
-
-            return {
-
-                // Image for temporary annotation page mockup
-                mockupImage: {
-                    filename: "annotation_mockup.jpg",
-                    height: 956,
-                    width: 1045
-                }
-            }
-        },
-
-        computed: {
-
-            ...mapState([
-
-                "pageData",
-                "pageOrder"
-            ])
-        }
-        
+// Allows for reference to store data by creating simple, implicit getters
+// Fields listed in mapState below can be found in the store (index.js)
+import { mapState } from "vuex";
+
+export default {
+  name: "Annotation",
+  data() {
+    return {
+      // TODO: "pages" is used as static testing data. Should be replaced with actual list of used categories
+      // from global state / store
+      pages: [
+        {title: 'Age', component: 'annot-age', id: 0},
+        {title: 'Sex', component: 'annot-gender', id: 1},
+        {title: 'Diagnosis', component: 'annot-diagnosis', id: 2},
+        {title: 'Assessment', component: 'annot-assessment', id: 3}
+      ],
     }
-
+  },
+  computed: {
+
+    ...mapState([
+      "pageData",
+      "pageOrder"
+    ])
+  },
+}
 </script>
 
 <style scoped>
```

Comment:
We should discuss how to bring the annotation page in line with the rest of the tool now. Do we want to address tool style consistency here? Or in a separate PR?

Reply:
What style do you mean? Coding style as in https://github.com/metaneuro/annotation_tool/pull/75 or css style?

Reply:
CSS style. Sorry - style tag seemed the best place to put this comment. Though in retrospect this kind of thing can go in the general 'Request Changes' comment box.

Reply:
I think separate issue and then PR is best for the CSS changeover. Like the gluing of the store to the data used by these components.